common/lib/capa/capa/responsetypes.py

```diff
@@ -1083,16 +1255,80 @@ def late_transforms(self, problem):
         self.do_shuffle(self.xml, problem)
         self.do_answer_pool(self.xml, problem)
 
-    def get_score(self, student_answers):
+    def grade_via_points(self, **kwargs):
```

Comment:
I'm trying to do a line by line comparison of the two pull requests, trying to figure out where they differ, and it seems like it's a lot more hard than it ought to be.